runsc/config/cgo.go

```diff
@@ -0,0 +1,3 @@
+package config
+
+var CgoImported bool
```

Comment:
Currently, in order to check whether CGO pkg is imported, we add a variable CgoImported to indicate this information, which will be updated during CGO pkg init() (defined in pkg/sentry/socket/plugin/cgo/stack_unsafe.go) and will be checked during preparing options for installing seccomp filters.
We wonder whether it is a proper way to do that or whether there is a more neat way to check if CGO is imported during runtime.

Reply:
You can use build constraints: https://stackoverflow.com/questions/65103958/is-it-possible-from-go-app-to-detect-if-cgo-enabled

Reply:
Good idea! Thanks for advice. We have updated the implementation with setting build constraints.